### PR TITLE
Fix SQL cmdlet New-AzSqlDatabaseExport to use network isolation parameters

### PR DESCRIPTION
**Pull request by brandong-ms**
## Description

During refactoring my cmdlets, I accidentally did not include the new parameters I am adding for network isolation in the export cmdlet. This is a fix for that issue, as well as adding some new test coverage and fixing a few tests in related areas. 

This does not have any API changes, just a bug and test fix. 

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Sql/Sql.Test/ScenarioTests/Common.ps1 [hunk 1]
```diff
@@ -808,12 +808,14 @@ function Get-SqlDatabaseImportExportTestEnvironmentParameters ($testSuffix)
     $exportBacpacUri = "http://test.blob.core.windows.net/bacpacs"
     $importBacpacUri = "http://test.blob.core.windows.net/bacpacs/test.bacpac"
     $storageKey = "StorageKey"
+	$storageResourceId = "/subscriptions/xys/resourcegroups/default/providers/Microsoft.Storage/test"
```
Comment (sredman) [inline src/Sql/Sql.Test/ScenarioTests/Common.ps1:811]:
```suggestion
    $storageResourceId = "/subscriptions/xys/resourcegroups/default/providers/Microsoft.Storage/test"
```
It looks like this line has an indentation problem

File: src/Sql/Sql.Test/ScenarioTests/Common.ps1 [hunk 2]
```diff
@@ -823,6 +825,9 @@ function Get-SqlDatabaseImportExportTestEnvironmentParameters ($testSuffix)
        }
        if ([System.string]::IsNullOrEmpty($storageKey)){
           throw "The  TEST_STORAGE_KEY environment variable should point to a valid storage key for an existing Azure storage account"
+       }
+	   if ([System.string]::IsNullOrEmpty($storageResourceId)){
```
Comment (sredman) [inline src/Sql/Sql.Test/ScenarioTests/Common.ps1:829]:
```suggestion
    if ([System.string]::IsNullOrEmpty($storageResourceId)){
```

File: src/Sql/Sql.Test/ScenarioTests/DatabaseBackupTests.ps1 [hunk 3]
```diff
@@ -79,26 +79,37 @@ function Test-RestoreGeoBackup
...
 	$restoredVcoreDbName = "powershell_db_deleted_vcore"
 
-	# this Get command has regression in MS when specifying Deletiondate. Fix should be in Prod by 5/7/2018. so currently use another way to do testing
-	$deletedDb = Get-AzSqlDeletedDatabaseBackup -ResourceGroupName $server.ResourceGroupName -ServerName $server.ServerName `
-		-DatabaseName $droppedDbName #-DeletionDate "2018-04-20 20:21:37.397Z" 
+	try
+	{
+		$server = Create-ServerForTest $rg $location
+	
+		# Create data warehouse database with all parameters.
```
Comment (sredman) [inline src/Sql/Sql.Test/ScenarioTests/DatabaseBackupTests.ps1:92]: I don't think this comment is accurate

File: src/Sql/Sql.Test/ScenarioTests/ImportExportTests.ps1 [hunk 4]
```diff
@@ -39,11 +54,26 @@ function Test-ImportNewDatabase
...
+    $useNetworkIsolation = $false
 
-    Verify-ImportExport $testSuffix $createServer $createDatabase $createFirewallRule $operationName $succeeded
+    Verify-ImportExport $testSuffix $createServer $createDatabase $createFirewallRule $operationName $succeeded $useNetworkIsolation 
 }
 
- function Verify-ImportExport($testSuffix, $createServer, $createDatabase, $createFirewallRule, $operationName, $succeeded)
+function Test-ImportNewDatabaseNetworkIsolation
+{
+	# Setup
```
Comment (sredman) [inline src/Sql/Sql.Test/ScenarioTests/ImportExportTests.ps1:64]:
```suggestion
    # Setup
```

File: src/Sql/Sql.Test/ScenarioTests/ImportExportTests.ps1 [hunk 5]
```diff
@@ -25,9 +25,24 @@ function Test-ExportDatabase
...
+    $succeeded = $true
+    $useNetworkIsolation = $false
    
-    Verify-ImportExport $testSuffix $createServer $createDatabase $createFirewallRule $operationName $succeeded
+    Verify-ImportExport $testSuffix $createServer $createDatabase $createFirewallRule $operationName $succeeded $useNetworkIsolation
+}
+
+function Test-ExportDatabaseNetworkIsolation
+{
+	# Setup
```
Comment (sredman) [inline src/Sql/Sql.Test/ScenarioTests/ImportExportTests.ps1:36]:
```suggestion
    # Setup
```

**Review by sredman — APPROVED** [review]

**Comment by VeryEarly** [comment]
Hi @brandong-ms , 

There is a CI fix in master branch please sync your code with master branch.
Also add changelog and resolve comments.

**Review by VeryEarly — DISMISSED** [review]

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**